src/Explorer/SplashScreen/SplashScreen.tsx

```diff
@@ -80,7 +75,13 @@ export class SplashScreen extends React.Component<SplashScreenProps> {
     const tipsItems = this.createTipsItems();
     const onClearRecent = this.clearMostRecent;
 
-    return (
+    const formContainer = (jsx: JSX.Element) => (
```

Comment:
Do we always want to show this? In the original code, it looks like we only show this when there's no tab open.